packages/react/src/alert-dialog/AlertDialog.tsx

```diff
@@ -0,0 +1,89 @@
+import * as RadixAlertDialog from "@radix-ui/react-alert-dialog";
+import { type ComponentPropsWithRef, type ReactNode, forwardRef } from "react";
+
+import type { ExtendProps } from "../utils";
+
+import { Box } from "../box";
+import { Button } from "../button";
+import { Flex } from "../flex";
+import * as styles from "./AlertDialog.css";
+
+type AlertDialogProps = ExtendProps<
+  ComponentPropsWithRef<typeof RadixAlertDialog.Root>,
+  ComponentPropsWithRef<typeof Box>,
+  {
+    action?: string;
+    actionAppearance?:
+      | "danger"
+      | "danger-outline"
+      | "default"
+      | "primary"
+      | "secondary";
```

Comment:
let's pick these instead of hard coding them
```suggestion
    actionAppearance?: ComponentPropsWithRef<typeof Button>["appearance"];
```